### PR TITLE
ITS: add ITS QC workflow with FHR in entire mode

### PR DESCRIPTION
**Pull request by JianLIUhep**

### CODE REVIEW

**Review by knopers8 — DISMISSED** [review]

File: scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+          "alio2-cr1-flp189",
+          "alio2-cr1-flp190",
+          "alio2-cr1-flp191",
+          "alio2-cr1-flp192",
+          "alio2-cr1-flp193",
+          "alio2-cr1-flp194",
+          "alio2-cr1-flp195",
+          "alio2-cr1-flp196",
+          "alio2-cr1-flp197",
+          "alio2-cr1-flp198"
```
Comment (vascobarroso) [inline scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json:89]: @JianLIUhep shouldn't `alio2-cr1-flp198` be removed from the list ?

File: scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
```
Comment (vascobarroso) [inline scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json:60]: @JianLIUhep shouldn't alio2-cr1-flp198 be removed from the list ?

File: scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json (hunk 1 above)
Comment (JianLIUhep) [reply scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json]: yes, done.

File: scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json (hunk 2 above)
Comment (JianLIUhep) [reply scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json]: done. thanks!

**Review by knopers8 — APPROVED** [review]

File: scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json (hunk 2 above)
Comment (knopers8) [reply scripts/etc/its-qcmn-fhr-fee-no-ds-entire.json]: it would not hurt, but at least will clear up any confusions.

**Pull request merged**